.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: benjamincharity/terminus-ui-ci
+      - image: getterminus/terminus-ui-circleci:12.9.1-20190905162749
```

Comment:
I thought this has been changed... maybe other repo?

Reply:
It was only changed for the UI lib originally as the Engage team updated it